package.json

```diff
@@ -8,7 +8,7 @@
     "start": "node production/index.js",
     "dev": "nodemon --exec babel-node src/index.js",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "test": "nyc --reporter=html --reporter=text mocha  --require  @babel/register  src/tests/*.js --exit",
+    "test": "export NODE_ENV=test && sequelize db:migrate:undo:all  && sequelize db:migrate && nyc --reporter=html --reporter=text mocha  --require  @babel/register  src/tests/*.js --timeout 10000 --exit",
```

Comment:
Remove the added commands and put them in a separate script instead. Also, ensure you make necessary changes in the Travis config file.